src/main/scala/intellij/haskell/psi/HaskellLayoutLexer.scala

```diff
@@ -79,24 +80,28 @@ class HaskellLayoutLexer(private val lexer: Lexer,
     var currentColumn = 0
 
     @tailrec
+    @inline
```

Comment:
Why did you inline here?

Reply:
Feel free to delete it. This was unnecessary.